nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,55 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 2.5.0
+***************
+
+:ref:`Core library <architecture>`
+==================================
+
+* The DFU result is now returned by the :c:param:`dfu_callback` callback in :c:struct:`nrf_modem_init_params` struct.
+* It is no longer necessary to call :c:func:`nrf_modem_init` twice when applying modem firmware updates.
+* Added the SoftSIM API to use a software SIM with the nRF modem.
+
+:ref:`Sockets <nrf_sockets>`
+============================
+
+* Added
+
+  * The :c:macro:`NRF_SO_EXCEPTIONAL_DATA` socket option to enable sending data as part of exceptional events (3GPP).
+  * The :c:macro:`MSG_WAITACK` flag to request a blocking send operation until the request is acknowledged by the network.
+  * Enhanced APN rate control.
+
+* Removed:
+
+  * The :c:param:`sa_len`, :c:param:`sin_len` and :c:param:`sin6_len` from the :c:struct:`nrf_sockaddr` :c:struct:`nrf_sockaddr_in` and :c:struct:`nrf_sockaddr_in6` structs, respectively.
+
+* Replaced the :c:macro:`NRF_SO_BINDTODEVICE` socket option with :c:macro:`NRF_SO_BINDTOPDN`. The new option takes an integer for the PDN ID.
+
+
+:ref:`AT interface <nrf_modem_at>`
+==================================
+
+* Added the option to set a timeout for AT commands by calling the :c:func:`nrf_modem_at_sem_timeout_set` function.
```

Comment:
We should rephrase this so that it is clearer that it's not a timeout for AT commands, but rather a timeout for how long AT commands shall wait for ongoing AT commands to complete.